modules/gdscript/gdscript.cpp

```diff
@@ -1652,13 +1674,22 @@ bool GDScriptInstance::get(const StringName &p_name, Variant &r_ret) const {
 		{
 			HashMap<StringName, GDScript::MemberInfo>::ConstIterator E = sptr->static_variables_indices.find(p_name);
 			if (E) {
-				if (E->value.getter) {
+				const GDScript::MemberInfo *static_member = &E->value;
+				if (static_member->getter) {
 					Callable::CallError ce;
-					r_ret = const_cast<GDScript *>(sptr)->callp(E->value.getter, nullptr, 0, ce);
-					return true;
+					if (static_member->getter_with_name) {
+						Variant property_name = p_name;
+						const Variant *args = &property_name;
+						r_ret = const_cast<GDScript *>(sptr)->callp(static_member->getter, &args, 1, ce);
+					} else {
+						r_ret = const_cast<GDScript *>(sptr)->callp(static_member->getter, nullptr, 0, ce);
+					}
+					if (ce.error == Callable::CallError::CALL_OK) {
+						return true;
+					}
 				}
-				r_ret = sptr->static_variables[E->value.index];
-				return true;
+				r_ret = sptr->static_variables[static_member->index];
+				return true; //index found
```

Comment:
```suggestion
				return true;
```

A nitpick, but this comment seems useless to me. Or it should follow the guidelines (begin with a capital letter and end with a dot). Also in this PR there are other whitespace changes that are undesirable in my opinion. But again, this is a minor remark.

Reply:
I'd agree that the removal of some empty lines are undesirable, probably considered useful for readability as they were kept when added originally